graphql/version_resolver.go

```diff
@@ -329,20 +329,11 @@ func (r *versionResolver) Tasks(ctx context.Context, obj *restModel.APIVersion,
 
 // TaskStatuses is the resolver for the taskStatuses field.
 func (r *versionResolver) TaskStatuses(ctx context.Context, obj *restModel.APIVersion) ([]string, error) {
-	defaultSort := []task.TasksSortOrder{
-		{Key: task.DisplayNameKey, Order: 1},
-	}
-	opts := task.GetTasksByVersionOptions{
-		Sorts:                          defaultSort,
-		IncludeBaseTasks:               false,
-		FieldsToProject:                []string{task.DisplayStatusKey},
-		IncludeBuildVariantDisplayName: false,
-	}
-	tasks, _, err := task.GetTasksByVersion(ctx, *obj.Id, opts)
+	statuses, err := task.GetTaskStatusesByVersion(ctx, *obj.Id)
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting version tasks: %s", err.Error()))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting task statuses for version with id '%s': %s", *obj.Id, err.Error()))
```

Comment:
I was under the impression that we started UI errors with "Error" because they actually are the errors being propagated to the user. Is this true? (We may want to shore up https://docs.google.com/document/d/1-DB1FQ_zb5fX_0pnVcwyU0HmJPrloFBu6qSE8dkk8PU/edit regardless)

Reply:
This is still a question but otherwise LGTM

Reply:
We aren't really consistent with this technically all errors appear in an Error toast which indicates that it is an error. I opened EVG-20187 to consolidate this.